planet/index.html

```diff
@@ -37,30 +36,76 @@
         <script type="text/javascript" src="js/GlobalCard.js"></script>
         <script type="text/javascript" src="js/ProjectViewer.js"></script>
         <script type="text/javascript" src="js/GlobalPlanet.js"></script>
-
         <script type="text/javascript" src="js/Planet.js"></script>
         <script type="text/javascript" src="js/main.js"></script>
+        <style>
+            .transparent-bg {
+              background-color: transparent !important;
+            }
```

Comment:
Any particular reason for internal css? can't we prefer external css? and why !important? can we play around specificity and make this work?